devenv/electrs/docker/Dockerfile

```diff
@@ -10,28 +10,23 @@ RUN apt-get install -qqy librocksdb-dev curl git
 FROM base as electrs-build
 RUN apt-get install -qqy cargo clang cmake
 
-ARG ELECTRS_NODE_VERSION='v0.10.0'
-ARG GIT_URI='https://github.com/romanz/electrs.git'
-ARG GIT_BRANCH='v0.10.0'
+ARG GIT_URI='https://github.com/mempool/electrs.git'
+ARG GIT_BRANCH='mempool'
```

Comment:
Is there a chance that this breaks the other services that use electrs?

Reply:
Might be, but I haven't seen any yet. I think it's still a fork and 90% compatible -- but I can't say for certain